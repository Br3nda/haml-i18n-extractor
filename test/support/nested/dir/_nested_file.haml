- content_for :title do
  = ['Admin', 'Accounts', 'New'].compact.join(' - ')
.row
  .col-md-12
    %ul.breadcrumb
      %li
        = link_to root_url(subdomain: 'admin') do
          %i.fa.fa-fw.fa-home
          %span.text Home
      %li
        = link_to admin_accounts_path do
          %i.fa.fa-fw.fa-globe
          %span.text New accounts
      %li.active=t('Create Account')

/ main
.content
  .main-header
    %h2 Create Account
  / main-content
  .main-content
    .row
      .col-md-12
        = render 'form'

%span.text Job ##{@job.id} (#{@job.queue})
