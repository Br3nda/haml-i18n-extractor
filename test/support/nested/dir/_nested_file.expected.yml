---
en:
  nested:
    dir:
      nested_file:
        admin: Admin
        accounts: Accounts
        new: New
        home: Home
        new_accounts: New accounts
        blah: BLAH
        blabbloo: BLABBLOO
        create_account: Create Account
        form: form
        job_jobid_jobqueue: 'Job #%{jobid} (%{jobqueue})'
